client/my-sites/site-settings/seo-settings/form.jsx

```diff
@@ -306,23 +311,25 @@ export class SeoForm extends React.Component {
 
 		const generalTabUrl = getGeneralTabUrl( slug );
 
-		const upsellProps = siteIsJetpack
-			? {
-					title: translate( 'Boost your search engine ranking' ),
-					feature: FEATURE_SEO_PREVIEW_TOOLS,
-					href: `/checkout/${ slug }/${ PRODUCT_UPSELLS_BY_FEATURE[ FEATURE_ADVANCED_SEO ] }`,
-			  }
-			: {
-					title: translate(
-						'Boost your search engine ranking with the powerful SEO tools in the Business plan'
-					),
-					feature: FEATURE_ADVANCED_SEO,
-					plan:
-						selectedSite.plan &&
-						findFirstSimilarPlanKey( selectedSite.plan.product_slug, {
-							type: TYPE_BUSINESS,
-						} ),
-			  };
+		const upsellProps =
+			siteIsJetpack && ! isAtomic
+				? {
+						title: translate( 'Boost your search engine ranking' ),
+						feature: FEATURE_SEO_PREVIEW_TOOLS,
+						href: `/checkout/${ slug }/${ PRODUCT_UPSELLS_BY_FEATURE[ FEATURE_ADVANCED_SEO ] }`,
+				  }
+				: {
+						title: translate(
+							'Boost your search engine ranking with the powerful SEO tools in the Business plan'
+						),
+						feature: FEATURE_ADVANCED_SEO,
+						plan: isFreeWPCOM
+							? PLAN_BUSINESS
```

Comment:
Is this because Atomic Free sites will be on the `jetpack_free` plan which `findFirstSimilarPlanKey` will fail for? If so, should this check be `isAtomic` to prevent us from changing the code run for Simple Free sites?

Reply:
Yes. Maybe that needs to be clearer.

I added `isAtomic`, so now it checks `isFreeWPCOM && isAtomic`. I think in general on Atomic plans we still want to check what term their plan is for, e.g. 2-year, 1-year, monthly to find the right plan slug.